brave/brave-core
# Fix 10182: Set SafeBrowsing endpoint at build time

Resolves https://github.com/brave/brave-browser/issues/10182

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
For developers: Before building please use the npmconfig specified here - https://github.com/brave/devops/wiki/npm-config-for-Brave-Developers

1. Open Brave Browser with a clean profile
2. Wait for ~5 mins and confirm <user-data-dir>/Safe Browsing has the *.store files
3. Navigate to https://testsafebrowsing.appspot.com - Confirm that webpage tests work.

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
